src/sci/configs/fulcro/routing/dynamic_routing.cljs

```diff
@@ -0,0 +1,89 @@
+(ns sci.configs.fulcro.routing.dynamic-routing
+  (:require [cljs.analyzer :as ana]
```

Comment:
Are you sure you want to pull in the CLJS analyzer for just throwing an error? It seems a waste of bundle size

Reply:
No, I do not. Good point, thx! Fixed now.